files/bootstrap.sh

```diff
@@ -331,3 +331,13 @@ fi
 systemctl daemon-reload
 systemctl enable kubelet
 systemctl start kubelet
+
+# Wait for the TLS certificate and private key to be present and add these to the config file so they are picked up the next time the kubelet starts.
+# These cannot be added upfront because they are not present when the kubelet first starts, and adding them would cause the kubelet to crash.
+TLS_CERT_FILE="/var/lib/kubelet/pki/kubelet-server-current.pem"
```

Comment:
TBH, i'm against this change, it's risky and ties to undefined behavior.
It works for current kubelet code, but if upstream kubelet changes behavior to be like "if tlsCertFile is specified, stop automatically rotate certificates", then we'll be break.

Additionally, tlsCertFile/tlsPrivateKeyFile flags are not actually used under current code[https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/util/certificate/certificate_store.go#L125], which means this is just an trick to make kube-bench happy. Instead, i'd prefer to change kube-bench to consider `serverTLSBootstrap` as secure as well

Reply:
A few of these changes made scare me, which is why I am never in favor of just making kube-bench happy, I think you are right we need to go and make some changes on that end.